lib/src/csv_parser.dart

```diff
@@ -101,14 +101,18 @@ class CSVParser {
         if (args.isNotEmpty) {
           strBuilder.write('namedArgs: {');
           for (final String arg in args) {
-            strBuilder.write('\'$arg\': $arg.toString(), ');
```

Comment:
since `arg` is dynamic it can be set to`null` event with NullSafety enabled, it is recommended to do interpolation instead of calling `toString()`.
Or at least check on null '?.toString()'.